src/test/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/model/confirmedarrival/ConfirmedArrivalServiceTest.kt

```diff
@@ -41,7 +41,9 @@ class ConfirmedArrivalServiceTest {
   @Test
   fun `add new confirmed arrival`() {
 
-    every { confirmedArrivalRepository.save(any()) } returns confirmedArrivalDb
+    every {
+      confirmedArrivalRepository.save(any())
+    } returns confirmedArrivalDb
```

Comment:
this fails for me locally due to type erasure - don't really understand why it fails for me but not on the build or for others.

Reply:
Looks okay locally for me, any odd behaviour from Intellij I always go to the 'Invalidate cache', fixes almost every odd local thing for me 😄